src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator start work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stop work");
```

Comment:
Similarly, the message 'Excavator stop work' should be more descriptive. You might want to change it to something like 'Excavator has stopped digging' to make it clear what action has been stopped.